src/CommandingOfficers/Tech.java

```diff
@@ -143,6 +158,8 @@ protected void perform(MapMaster gameMap)
       // Overcharge
       for(Unit u: myCommander.units)
       {
+        // Only mechanical units get overcharged.
+        if( u.model.isNone(UnitModel.TROOP) )
         u.alterHP(healAmount, true);
```

Comment:
Technically different rules here :P

Reply:
Different than what?

Reply:
Your stat boosts also require the unit to be one of `TANK`, `SHIP`, `HOVER`, or `JET` in addition to not being `TROOP`.

Reply:
True I suppose. Is there any units who are classified differently in the two places? I guess I should make it consistent anyway, for form's sake.

Reply:
I don't think so, but it's not out of the question.
IMO the real test *should* just be "not `TROOP`", since `TROOP` is how we indicate the "form factor" of the unit is humanish.
...which makes it weird because I kind of assumed that I wouldn't classify Wargroove Knights as `TROOP`. I guess it's relative, lol.

Reply:
I guess we can figure that out if/when we add Wargroove units.